code/modules/supermatter/supermatter.dm

```diff
@@ -500,14 +513,14 @@
 	Consume(AM)
 
 
-/obj/machinery/power/supermatter/proc/Consume(var/mob/living/user)
+/obj/machinery/power/supermatter/proc/Consume(var/mob/living/user, var/amount = 200)
 	if(istype(user))
 		user.dust()
-		power += 200
+		power += amount
 	else
 		qdel(user)
 
-	power += 200
+	power += amount
```

Comment:
Setting the amount of power added per call param is a bit odd. Beats Random Magic Numbers though.